Gemfile

```diff
@@ -35,7 +39,7 @@ group(:development, :test) do
   # be removed here *yet* due to TravisCI / AppVeyor which call:
   # bundle install --without development
   # PUP-7433 describes work necessary to restructure this
-  gem "rake", "10.1.1", :require => false
+  gem "rake", '~> 12.3', :require => false
```

Comment:
@joshcooper updated. I chose this version because that is what is required by packaging.